depends/funcs.mk

```diff
@@ -264,7 +264,8 @@ $(foreach package,$(packages),$(eval $(package)_type=$(host_arch)_$(host_os)))
 $(foreach package,$(all_packages),$(eval $(call int_vars,$(package))))
 
 #include package files
-$(foreach package,$(all_packages),$(eval include packages/$(package).mk))
+$(foreach package,$(native_packages),$(eval include packages/$(package).mk))
```

Comment:
To match above?
```suggestion
$(foreach native_package,$(native_packages),$(eval include packages/$(native_package).mk))
```

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/23673#issuecomment-990706374).